.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 .soroban/
+.DS_Store
```

Comment:
You may want to put this into a global gitignore so that by default you don't find these files in your git repos. I use that and have found it handy: https://github.com/leighmcculloch/dotfiles/blob/c490042da914da097a856ef0281e3d17a3debc04/files/gitconfig#L7